### PR TITLE
Add Chinese Translation for ChangeLog.md

### PR DESCRIPTION
**Pull request by SpZanG**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #340 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-zhcn/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# 更新日志
+
+此文件由{{gitbook.time}}自动生成
+
+## V1.2-RC 2019年10月5号 - 预发行版 (仅限英语)
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
```
Comment (boblone19) [inline Document-zhcn/CHANGELOG.md]: 晋升为同类安全指南最成功的作品

Comment (sushi2k) [inline Document-zhcn/CHANGELOG.md]: Please always use suggestion function:

```suggestion
晋升为同类安全指南最成功的作品。
```

Comment (SpZanG) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 晋升为同类安全指南最成功的作品。
```

File: Document-zhcn/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- 删除要求 2.6 "在可能包含敏感信息的文字栏上停止使用剪贴板"。
+- 增加要求 2.2 "任何敏感资料不可以被存储在应用程序容器之外或者系统凭据存储设施中（System credential storage facilities）"。
+- 更新要求 2.1 的用词，改为 "系统凭据存储设施可以适当的用于存储敏感信息，例如：个人资料，使用者凭据或者加密密钥"。
+
+## V1.0 2018年1月12号
+
+以下为1.0版本所包含的更新内容：
+
+- 因为要求 8.9 和要求 8.12 相同，因此删除要求 8.9。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 因为要求8.9和要求 8.12 相同，因此删除要求8.9。
```

File: Document-zhcn/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- 修正要求7.1内西班牙文翻译问题。
+- 在致谢区重新安排翻译者名字。
+
+## V1.1.2 2019年1月3号 - 赞助者与国际翻译版
+
+以下为1.1.2版本所包含的更新内容：
+
+- 新增感谢信，这封感谢信给予那些购买了电子版的人。
+- 在V4中添加了丢失身份验证链接和更新损坏的身份验证。
+- 修正英文版中的 4.7 和 4.8 互换。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 修正英文版中的4.7和4.8互换。
```

File: Document-zhcn/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- 更新法文和西班牙文翻译。
+- 完成更新日志的繁体中文和日文翻译。
+- 自动验证 轻量级标记语言 的语法以及网址是否可以访问。
+- 新增标记代码到各个要求，以便于未来在MSTG中可以快速以及方便的找到相关建议和测试方案。
+- 缩减仓库大小，新增Generated到.gitignore文件中。
+- 新增行为守则以及撰稿准则（Contributing guidelines）。
+- 新增拉取要求（Pull-Request）模版。
+- 更新与Gitbook网页仓库同步的功能。
+- 更新所有翻译版本用来产生 XML/JSON/CSV 的脚本。
+- 完成繁体中文版 前言 的翻译。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 完成繁体中文版《前言》的翻译。
```

File: Document-zhcn/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- 修复所有 轻量级标记语言（markdown）问题。
+- 更新法文和西班牙文翻译。
+- 完成更新日志的繁体中文和日文翻译。
+- 自动验证 轻量级标记语言 的语法以及网址是否可以访问。
+- 新增标记代码到各个要求，以便于未来在MSTG中可以快速以及方便的找到相关建议和测试方案。
+- 缩减仓库大小，新增Generated到.gitignore文件中。
+- 新增行为守则以及撰稿准则（Contributing guidelines）。
+- 新增拉取要求（Pull-Request）模版。
+- 更新与Gitbook网页仓库同步的功能。
+- 更新所有翻译版本用来产生 XML/JSON/CSV 的脚本。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 更新所有翻译版本用来产生XML/JSON/CSV 的脚本。
```

File: Document-zhcn/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+- 增加要求 MSTG‑ARCH‑12，以向应用程序开发人员展示他们应遵守的相关的国际隐私法律。
+- 为英文版的引用（references）创建了一个固定的格式。
+- 添加要求 MSTG‑PLATFORM‑11，以反击通过第三方键盘进行的间谍活动（spying）。
+- 添加要求 MSTG-MSTG‑RESILIENCE‑13，以阻止在应用程序中的窃听。
+
+## V1.1.4 2019年7月4号 - 峰会版
+
+以下为1.1.4版本所包含的更新内容：
+
+- 修复所有 轻量级标记语言（markdown）问题。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 修复所有轻量级标记语言（markdown）问题。
```

File: Document-zhcn/CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
+- 添加要求 MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
+- 在怎样使用MASVS进行风险评估中添加了一点描述。
+- 在付费内容（paid content）中添加了一点描述。
+- 增加要求 MSTG‑ARCH‑11，以包含针对L2应用程序的 责任披露政策（Responsible Disclosure policy）。
+- 增加要求 MSTG‑ARCH‑12，以向应用程序开发人员展示他们应遵守的相关的国际隐私法律。
+- 为英文版的引用（references）创建了一个固定的格式。
+- 添加要求 MSTG‑PLATFORM‑11，以反击通过第三方键盘进行的间谍活动（spying）。
+- 添加要求 MSTG-MSTG‑RESILIENCE‑13，以阻止在应用程序中的窃听。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 添加要求MSTG-MSTG‑RESILIENCE‑13，以阻止在应用程序中的窃听。
```

File: Document-zhcn/CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
+- 添加要求 MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
+- 在怎样使用MASVS进行风险评估中添加了一点描述。
+- 在付费内容（paid content）中添加了一点描述。
+- 增加要求 MSTG‑ARCH‑11，以包含针对L2应用程序的 责任披露政策（Responsible Disclosure policy）。
+- 增加要求 MSTG‑ARCH‑12，以向应用程序开发人员展示他们应遵守的相关的国际隐私法律。
+- 为英文版的引用（references）创建了一个固定的格式。
+- 添加要求 MSTG‑PLATFORM‑11，以反击通过第三方键盘进行的间谍活动（spying）。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 添加要求MSTG‑PLATFORM‑11，以反击通过第三方键盘进行的间谍活动（spying）。
```

File: Document-zhcn/CHANGELOG.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
+- 添加要求 MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
+- 在怎样使用MASVS进行风险评估中添加了一点描述。
+- 在付费内容（paid content）中添加了一点描述。
+- 增加要求 MSTG‑ARCH‑11，以包含针对L2应用程序的 责任披露政策（Responsible Disclosure policy）。
+- 增加要求 MSTG‑ARCH‑12，以向应用程序开发人员展示他们应遵守的相关的国际隐私法律。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 增加要求MSTG‑ARCH‑12，以向应用程序开发人员展示他们应遵守的相关的国际隐私法律。
```

File: Document-zhcn/CHANGELOG.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
+- 添加要求 MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
+- 在怎样使用MASVS进行风险评估中添加了一点描述。
+- 在付费内容（paid content）中添加了一点描述。
+- 增加要求 MSTG‑ARCH‑11，以包含针对L2应用程序的 责任披露政策（Responsible Disclosure policy）。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 增加要求MSTG‑ARCH‑11，以包含针对L2应用程序的 责任披露政策（Responsible Disclosure policy）。
```

File: Document-zhcn/CHANGELOG.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
+- 添加要求 MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 添加要求MSTG‑AUTH‑12，以提醒开发人员部署授权，特别是在多用户应用程序（multi-user apps）的情况下。
```

File: Document-zhcn/CHANGELOG.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+## V1.2-RC 2019年10月5号 - 预发行版 (仅限英语)
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
+- 添加要求 MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 添加要求MSTG‑PLATFORM‑10，以进一步安全的使用WebViews。
```

File: Document-zhcn/CHANGELOG.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+## V1.2-RC 2019年10月5号 - 预发行版 (仅限英语)
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
+- 更新要求 MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 更新要求MSTG‑CODE‑4，以涵盖更多信息而不仅仅是排错调试（debugging）。
```

File: Document-zhcn/CHANGELOG.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+此文件由{{gitbook.time}}自动生成
+
+## V1.2-RC 2019年10月5号 - 预发行版 (仅限英语)
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
+- 增加侧重于数据保护的要求 MSTG-STORAGE-13， MSTG-STORAGE-14， 和 MSTG-STORAGE-15。
+- 更新要求 MSTG-AUTH-11，以保护上下文信息（contextual information）。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 更新要求MSTG-AUTH-11，以保护上下文信息（contextual information）。
```

File: Document-zhcn/CHANGELOG.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
+# 更新日志
+
+此文件由{{gitbook.time}}自动生成
+
+## V1.2-RC 2019年10月5号 - 预发行版 (仅限英语)
+
+以下为1.2-RC版本所包含的更新内容：
+
+- 晋升到旗舰地位。
+- 更改要求 MSTG-STORAGE-1 "需要被使用"（need to be used）。
```
Comment (jack-smith-dh) [inline Document-zhcn/CHANGELOG.md]:
```suggestion
- 更改要求MSTG-STORAGE-1 "需要被使用"（need to be used）。
```

**Comment by jack-smith-dh** [comment]
reviewed, good to proceed

**Pull request merged**